### PR TITLE
1890 Authsession with assertion

### PR DESCRIPTION
**Pull request by Daniel-Echegaray**

### CODE REVIEW

File: Projects/AccountSDKIOSWeb/Tests/AccountSDKIOSWebTests/URLBuilderTests.swift [hunk 1]
```diff
@@ -66,7 +66,7 @@ final class URLBuilderTests: XCTestCase {
     
     func testLoginURLWithMFAIncludesACRValues() {
         let sut = URLBuilder(configuration: Fixtures.clientConfig)
-        let authRequest = URLBuilder.AuthorizationRequest(loginHint: "", extraScopeValues: [])
+        let authRequest = URLBuilder.AuthorizationRequest(loginHint: "", assertion: "", extraScopeValues: [])
         let loginURL = sut.loginURL(authRequest: authRequest, authState: AuthState(mfa: .otp))
         
         XCTAssertEqual(loginURL?.scheme, "https")
```
Comment (zamorastp) [inline Projects/AccountSDKIOSWeb/Tests/AccountSDKIOSWebTests/URLBuilderTests.swift:72]: is it valuable to add test with assertion?

**Review by zamorastp — APPROVED** [review]

File: Sources/AccountSDKIOSWeb/Client/Client.swift [hunk 2]
```diff
@@ -115,14 +115,15 @@ public class Client: CustomStringConvertible {
         return authState
     }
     
-    private func createWebAuthenticationSession(withMFA: MFAType? = nil,
-                                                loginHint: String?,
-                                                extraScopeValues: Set<String> = [],
-                                                completion: @escaping LoginResultHandler) -> ASWebAuthenticationSession {
+    func createWebAuthenticationSession(withMFA: MFAType? = nil,
+                                        loginHint: String?,
```
Comment (zamorastp) [inline Sources/AccountSDKIOSWeb/Client/Client.swift:119]: for consistency can we provide default value for loginHint?

Comment (Daniel-Echegaray) [reply Sources/AccountSDKIOSWeb/Client/Client.swift:119]: Sure I will add it

File: Projects/AccountSDKIOSWeb/Tests/AccountSDKIOSWebTests/URLBuilderTests.swift (hunk 1 above)
Comment (Daniel-Echegaray) [reply Projects/AccountSDKIOSWeb/Tests/AccountSDKIOSWebTests/URLBuilderTests.swift:99]: I added a unit test for assertion string 🚀

**Pull request merged**